test/pipe.js

```diff
@@ -11,13 +11,6 @@ shell.mkdir('tmp');
 // Invalids
 //
 
-// no piped methods for commands that don't return anything
-assert.throws(function() {
```

Comment:
Almost every command returns a ShellString now. So the number of "commands that don't return anything" has shrunk to almost zero.